samples/dynamodb-geo-server/pom.xml

```diff
@@ -103,6 +103,37 @@
 						</webResources>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.tomcat.maven</groupId>
+					<artifactId>tomcat7-maven-plugin</artifactId>
+					<version>2.2</version>
+					<configuration>
+						<!-- http port -->
+						<port>9090</port>
+						<!-- application path always starts with /-->
+						<path>/</path>
+						<!-- optional path to a context file -->
+						<contextFile>${tomcatContextXml}</contextFile>
+						<!-- optional system propoerties you want to add -->
+                        <systemProperties>
+							<!--
+                            <AWS_ACCESS_KEY_ID>AKIAJBG2H2TJ5DSDOS6A</AWS_ACCESS_KEY_ID>
```

Comment:
Wow. Yeah, I got an email from was and did that. Rookie mistake. Thanks. Seems like for local use you can put any string in there anyway.